keyboards/handwired/datahand/rules.mk

```
@@ -0,0 +1,59 @@
+# Project-specific includes
+SRC = matrix.c
+
+# MCU name
+MCU = AT90USB1286
```

The travis CI error is occurring because of this, I think.

This should be lower case, and yes that does matter.

```suggestion
MCU = at90usb1286
```